.github/workflows/main_push_and_pull_request_workflow.yml

```diff
@@ -10,17 +10,17 @@ jobs:
     strategy:
       matrix:
         java-version: [11, 17]
-        runs-on: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
aligning with other Connectors, unless there's a reason to test against multiple OSs.